pkg/cluster/cluster.go

```diff
@@ -78,11 +78,11 @@ func CreateCluster(ctx context.Context, cluster *k3d.Cluster, runtime k3drt.Runt
 	}
 	cluster.Network.Name = networkID
 	extraLabels := map[string]string{
-		"k3d.cluster.network":          networkID,
-		"k3d.cluster.network.external": strconv.FormatBool(cluster.Network.External),
+		k3d.NetworkLabelName:         networkID,
```

Comment:
Good thing moving the label names to types :+1: 
This was on the list anyway :)
I'm just not sure, if we'd be better off going with `LabelXYZ` (e.g. `LabelNetwork`, `LabelNetworkExternal`), i.e. with the `Label` as prefix to easier find it when using lookups in IDEs (mostly for IDEs that don't do fuzzy lookups/searches) :thinking:

Reply:
yes rename in commit 18811fe